CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 ### Bug fixes
 * [Object Server] Fixed a bug where long reconnection happens when a proxy in front of the sync worker returns one of those.
+* Fix a bug where `Realm.open` could unexpectedly raise a "Realm at path ... already opened with different schema version" error.
```

Comment:
This has to be added to a 2.1.2 section, as 2.1.1 was already released.

Reply:
Ugh, I’m used to empty change log sections being added after a release is made, so I didn’t even notice this section was for a version that was already released.

Reply:
i will handle it